modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -374,65 +374,69 @@ gcloud projects add-iam-policy-binding <service-project-id> \
 ```
 ====
 
-* Bind GKE service account with the Workload Identity role
+. Bind the service accounts.
 +
-.Show commands
+The account ID of the GCP service account is used to configure service account findings. This is the email prefix of the GCP service account.
+For example, if the GCP service account is `my-gcp-sa@my-project.iam.gserviceaccount.com`, then the account ID is `my-gcp-sa`.
++
+* Redpanda cluster service account
++
+.Show command
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-gke@<service-project-id>.iam.gserviceaccount.com \
-    --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
-gcloud iam service-accounts add-iam-policy-binding redpanda-gke@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-cluster-gcp-sa-account-id>]"
```

Comment:
Is t necessary to have such a long placeholder? We use `<service-account-name>` elsewhere. Could this be `<service-account-id>`?